library/oidc.c

```diff
@@ -97,10 +107,16 @@ static void dump_cb(tlsuv_http_req_t *r, char *data, ssize_t len) {
     } else {
         oidc_req *req = r->data;
         fprintf(stderr, "status = %zd\n", len);
-        complete_oidc_req(req, (int)len, NULL);
+      //  complete_oidc_req(req, (int)len, NULL);
+    }
+}
+static void dump_resp(tlsuv_http_resp_t *resp) {
```

Comment:
why printf not ZITI_LOG? is it because it's on a different thread/process? does anything make it into the logs for review after the fact? anything sensitive logged?